src/platform/user/profile/vet360/util/local-vet360.js

```diff
@@ -113,6 +113,8 @@ export const mockContactInformation = {
     countryCodeFips: 'US',
     countryCodeIso2: 'US',
     countryCodeIso3: 'USA',
+    latitude: 37.5615,
```

Comment:
These fields should be coming through for VA Profile addresses, so I thought it made sense to add.